### DISCUSSION TITLE
First release of cargo-3ds

### DISCUSSION THREAD

**Discussion by Meziu** [rust3ds/ctru-rs#132] [Announcements]
[`cargo-3ds`](https://github.com/rust3ds/cargo-3ds) version `0.1.0` has been finally released on [crates.io](https://crates.io/crates/cargo-3ds). The @rust3ds/active team has been appointed as team-owners of the package.

Let's celebrate 🎊 🥳

**Comment by ian-h-chamberlain** [comment]
Woo, awesome! Sorry I haven't been as active recently, my spare time is no longer as plentiful as it once was. Thanks for doing the work to get a release together and make some meaningful progress on this project!!

**Comment by AzureMarker** [comment]
Congrats on the release! :tada: It's great to see things start to stabilize. I've also been very busy and haven't had much spare time... But still trying to keep up to date on what's going on!

**Comment by Techie-Pi** [comment]
Amazing to see things being released! 🎊 I've (also) been quite busy and I don't have much spare time 😅 Still trying to keep up tho!
